data/api/application-service/definitions/registration.yaml

```diff
@@ -29,7 +29,9 @@ properties:
     description: A secret token that the homeserver will use authenticate requests to the application service.
   sender_localpart:
     type: string
-    description: The localpart of the user associated with the application service.
+    description: |-
+      The localpart of the user associated with the application service. Events will be sent to the AS if this user is the target of the event, or
+      is a joined member of the room where the event occurred.
```

Comment:
Would it just be better to change the `users` description to be:

"...if a local user matching one of the the namespaces or the `sender_localpart` is the target..."

Reply:
It seems a bit strange to me, if the only place that this behaviour of `sender_localpart` is defined is in `namespaces` (which you might not even use).

Reply:
yes, but I meant we should update `namespaces.users` *as well as* `sender_localpart`.

Reply:
> Ok so if I understand correctly you want to do your change in addition to the one that is already in this pull request, right?

Yes, right.

> If so do you think it would be better to switch out your suggestion with [my first reply](https://github.com/matrix-org/matrix-spec/pull/1810#discussion_r1585218528), while still keeping my initial change (contents of this PR)?

Oh right. Er well, not really? But I don't feel that strongly about it. It would probably be easier if you just made a change and we can see how it looks, before discussing further.